### DISCUSSION TITLE
Autocomplete for shortcuts?

### DISCUSSION THREAD

**Discussion by lunelson** [unocss/unocss#1012] [Ideas]
I was wondering how to change the pattern for certain rules that are built in to presets, and it seems shortcuts would be a good way to alias them—however it also seems that autocomplete would be lost. 

I was hoping that something like this would work:

```js
[/^bg-mode-(.+)$/, ([, mode]) => `bg-blend-${mode}`, { autocomplete: ['bg-mode-(color|normal)'] }]
```

**Comment by antfu** [comment]
Fixed in 0.35.1 Thanks.
